packages/venia-ui/lib/components/AccountMenu/accountMenu.css

```diff
@@ -1,30 +1,57 @@
 .root {
+    --content-width: 27.5rem;
+
+    height: 0;
+    overflow: visible;
+    left: -100vw;
+    opacity: 0;
+    position: absolute;
+    top: 100%;
+    transition: opacity 192ms var(--venia-global-anim-out),
+        visibility 192ms var(--venia-global-anim-out),
+        /* Adding delay to move container off the screen after previous transitions happened */
+            left 0s 192ms;
```

Comment:
Weird indention here. Is this prettier? Or can we pull it back a tab?

Reply:
Yes, this is formatted by prettier.